docs/core/tools/custom-templates.md

```diff
@@ -13,6 +13,9 @@ You can install custom templates from a NuGet package on any NuGet feed, by refe
 
 The template engine is open source, and the online code repository is at [dotnet/templating](https://github.com/dotnet/templating/) on GitHub. Visit the [dotnet/dotnet-template-samples](https://github.com/dotnet/dotnet-template-samples) repo for samples of templates. More templates, including templates from third parties, are found at [Available templates for dotnet new](https://github.com/dotnet/templating/wiki/Available-templates-for-dotnet-new) on GitHub. For more information about creating and using custom templates, see [How to create your own templates for dotnet new](https://devblogs.microsoft.com/dotnet/how-to-create-your-own-templates-for-dotnet-new/) and the [dotnet/templating GitHub repo Wiki](https://github.com/dotnet/templating/wiki).
 
+> [!NOTE]
```

Comment:
How about removing the templating repo from the preceding paragraph and introducing it in the note as an archived resource that might still have something useful?

Reply:
Ack. I just realized I linked to the wrong repo. Only the samples repo isn't maintained. But yes, I like your idea.